src/BodyParams/BodyParamsMiddleware.php

```diff
@@ -82,10 +83,17 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
             }
 
             // Matched! Parse and pass on to the next
-            return $next(
-                $strategy->parse($request),
-                $response
-            );
+            try {
+                return $next(
+                    $strategy->parse($request),
+                    $response
+                );
+            } catch (MalformedRequestBodyException $exception) {
```

Comment:
@weierophinney I agree with @danizord's suggestion (thanks!) It satisfies my concern while not creating more work for the 80% of folks who don't need it. I'll make the adjustments.